src/components/SearchField/SearchField.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+
+export class SearchField extends React.Component {
+  state = {
+    query: '',
+  };
+
+  render() {
+    const { movies } = this.props;
+
+    const searchQuery = this.state.query.toLocaleLowerCase().trim();
+    const filteredMovies = [...movies]
+      .filter(movie => movie.title.toLocaleLowerCase().includes(searchQuery)
```

Comment:
```suggestion
    const filteredMovies = movies.filter(movie => 
      movie.title.toLocaleLowerCase().includes(searchQuery)
```
You have no need to create the copy of 'movies' because 'filter' is a non-mutating method